woocommerce-services.php

```diff
@@ -879,7 +888,7 @@ public function attach_hooks() {
 			$this->paypal_ec->init();
 
 			// Only register shipping label-related logic if WC Shipping is not active.
-			if ( ! self::is_wc_shipping_activated() ) {
+			if ( WC_Connect_Options::get_option( 'legacy_site' ) ) {
```

Comment:
Do we need to keep `! self::is_wc_shipping_activated()` in place since we still need parallel support with WC Shipping for legacy sites?

_Actually, I think this could create conflicts for legacy sites if WC Shipping is also activated since they try to register the same endpoints?_

Reply:
Hmmm, not 100% I'll test it out and report back.